doc/index.txt

```diff
@@ -168,6 +168,60 @@ Community
 
 * Come visit us in Montreal! Most developers are students in the LISA_ group at the `University of Montreal`_.
 
+
+.. _support:
+
+Help!
+=====
```

Comment:
That's a good section to have, thanks!
I think it would be worth having an explicit point about looking for the answer to similar questions before looking for help.